index.test-d.ts

```diff
@@ -0,0 +1,5 @@
+import {expectType} from 'tsd-check';
+import appPath from '.';
+
+expectType<Promise<string>>(appPath('Safari'));
```

Comment:
You can use `expectType<string>(await appPath('Safari'));`

Reply:
No, I want to make sure that a promise is returned. It's possible to use `await` on non-promise types which simply wraps the type in a promise. Thus with `await` I can miss a possible bug.

Reply:
That's a very good point. I never thought about that.